doc/model_parameter.md

```diff
@@ -158,6 +158,11 @@ INTO sqlflow_models.my_xgb_regression_model;
 	<td>float32</td>
 	<td>Subsample ratio of the training instance.</td>
 </tr>
+<tr>
+	<td>train.cache</td>
```

Comment:
Attributes start with `train.` is not used for model definitions, can this be defined only in codegen?